bin/predictions.py

```diff
@@ -35,31 +35,46 @@
 from catlas.sankey.sankey_utils import Sankey
 
 
-# Load inputs and define global vars
-if __name__ == "__main__":
-    """Run predictions according to input config file
+def parse_inputs():
```

Comment:
I think we should move all of these upfront function definitions to another util file